docs/react-v9/contributing/using-local-unpublished-version-of-the-lib-with-a-local-React-app.md

```diff
@@ -0,0 +1,111 @@
+In addition to the standard [development practice](https://github.com/microsoft/fluentui/wiki/Development-Workflow), it is sometimes useful to run a local app against the introduced changes within the library, for instance, when trying to repro a bug locally, etc. In this case, `yarn link` tool comes in real handy, and this short guide will take you through the full setup to get you up and running.
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31759&commitId=29b962026e882a6eadfb46703c4b22e38431ea24&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Size - Dark Mode.normal.chromium.png|2| Changed|
|Accordion.Size.normal.chromium.png|2| Changed|
|Accordion.Size - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Avatar</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Colorful.normal.chromium.png|325| Changed|
|Avatar.Colorful - Dark Mode.normal.chromium.png|452| Changed|
|Avatar.Colorful - RTL.normal.chromium.png|326| Changed|
|Avatar.Active Appearance.normal.chromium.png|10| Changed|
|Avatar.Active.normal.chromium.png|5| Changed|
|Avatar.Color.normal.chromium.png|1| Changed|
|Avatar.Active Appearance - Dark Mode.normal.chromium.png|2| Changed|
|Avatar.Color - RTL.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Badge</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge.Appearance.normal.chromium.png|1| Changed|
|Badge.Shape.normal.chromium.png|1| Changed|
|Badge.Appearance - Dark Mode.normal.chromium.png|15| Changed|
|Badge.Default.normal.chromium.png|1| Changed|
|Badge.Color.normal.chromium.png|1| Changed|
|Badge.Size.normal.chromium.png|6| Changed|

</details>


  <details><summary><b>Button</b> 75 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Large Icon Only - Dark Mode.default.chromium.png|4| Changed|
|Button.Large Icon Only.default.chromium.png|5| Changed|
|Button.Large With Icon - Dark Mode.default.chromium.png|782| Changed|
|Button.Large Icon Only.pressed.chromium.png|3| Changed|
|Button.Large Icon Only - Dark Mode.pressed.chromium.png|5| Changed|
|Button.Large With Icon - Dark Mode.hover.chromium.png|731| Changed|
|Button.Large Icon Only.hover.chromium.png|4| Changed|
|Button.Large With Icon - RTL.hover.chromium.png|761| Changed|
|Button.Large With Icon.default.chromium.png|795| Changed|
|Button.Large With Icon.hover.chromium.png|761| Changed|
|Button.Large Icon Only - Dark Mode.hover.chromium.png|3| Changed|
|Button.Large With Icon - RTL.pressed.chromium.png|707| Changed|
|Button.Medium With Icon - RTL.hover.chromium.png|753| Changed|
|Button.Medium With Icon - Dark Mode.hover.chromium.png|720| Changed|
|Button.Large With Icon - RTL.default.chromium.png|795| Changed|
|Button.Medium With Icon - Dark Mode.default.chromium.png|775| Changed|
|Button.Outline Disabled - Dark Mode.hover.chromium.png|159| Changed|
|Button.Medium With Icon.pressed.chromium.png|697| Changed|
|Button.Outline Disabled - Dark Mode.pressed.chromium.png|159| Changed|
|Button.Large With Icon - Dark Mode.pressed.chromium.png|822| Changed|
|Button.Medium With Icon - RTL.default.chromium.png|788| Changed|
|Button.Medium With Icon - Dark Mode.pressed.chromium.png|813| Changed|
|Button.Outline Disabled Focusable - Dark Mode.pressed.chromium.png|159| Changed|
|Button.Outline Disabled Focusable - Dark Mode.default.chromium.png|159| Changed|
|Button.Primary Disabled - Dark Mode.default.chromium.png|44| Changed|
|Button.Medium With Icon - RTL.pressed.chromium.png|697| Changed|
|Button.Primary Disabled - Dark Mode.pressed.chromium.png|44| Changed|
|Button.Primary Disabled - Dark Mode.hover.chromium.png|44| Changed|
|Button.Outline Disabled - Dark Mode.default.chromium.png|159| Changed|
|Button.Primary Disabled Focusable.hover.chromium.png|15| Changed|
|Button.Medium With Icon.default.chromium.png|788| Changed|
|Button.Large With Icon.pressed.chromium.png|707| Changed|
|Button.Primary Disabled Focusable - Dark Mode.default.chromium.png|44| Changed|
|Button.Medium With Icon.hover.chromium.png|753| Changed|
|Button.Primary Disabled Focusable - Dark Mode.hover.chromium.png|44| Changed|
|Button.Primary Disabled.hover.chromium.png|15| Changed|
|Button.Primary Disabled Focusable - Dark Mode.pressed.chromium.png|44| Changed|
|Button.Primary Disabled Focusable.pressed.chromium.png|15| Changed|
|Button.Primary Disabled.pressed.chromium.png|15| Changed|
|Button.Small With Icon - Dark Mode.hover.chromium.png|311| Changed|
|Button.Small With Icon - RTL.pressed.chromium.png|301| Changed|
|Button.Outline Disabled Focusable - Dark Mode.hover.chromium.png|159| Changed|
|Button.Small With Icon - RTL.hover.chromium.png|324| Changed|
|Button.Small With Icon.hover.chromium.png|324| Changed|
|Button.Small With Icon - RTL.default.chromium.png|339| Changed|
|Button.Small With Icon - Dark Mode.pressed.chromium.png|351| Changed|
|Button.Subtle Disabled Focusable.default.chromium.png|25| Changed|
|Button.Subtle Disabled Focusable - Dark Mode.hover.chromium.png|187| Changed|
|Button.Subtle Disabled - Dark Mode.default.chromium.png|187| Changed|
|Button.Small With Icon.default.chromium.png|339| Changed|
|Button.Subtle Disabled Focusable.hover.chromium.png|25| Changed|
|Button.Subtle Disabled - Dark Mode.hover.chromium.png|187| Changed|
|Button.Primary Disabled Focusable.default.chromium.png|15| Changed|
|Button.Subtle Disabled.hover.chromium.png|25| Changed|
|Button.Subtle Disabled Focusable - Dark Mode.pressed.chromium.png|187| Changed|
|Button.Subtle Disabled.default.chromium.png|25| Changed|
|Button.Transparent Disabled - Dark Mode.default.chromium.png|199| Changed|
|Button.Subtle Disabled Focusable.pressed.chromium.png|25| Changed|
|Button.Transparent Disabled Focusable - Dark Mode.hover.chromium.png|199| Changed|
|Button.Small With Icon.pressed.chromium.png|301| Changed|
|Button.Subtle Disabled Focusable - Dark Mode.default.chromium.png|187| Changed|
|Button.Primary Disabled.default.chromium.png|15| Changed|
|Button.Transparent Disabled Focusable - Dark Mode.pressed.chromium.png|199| Changed|
|Button.Small With Icon - Dark Mode.default.chromium.png|334| Changed|
|Button.Transparent Disabled Focusable.hover.chromium.png|27| Changed|
|Button.Transparent Disabled Focusable.default.chromium.png|27| Changed|
|Button.Transparent Disabled Focusable.pressed.chromium.png|27| Changed|
|Button.Transparent Disabled Focusable - Dark Mode.default.chromium.png|199| Changed|
|Button.Transparent Disabled.hover.chromium.png|27| Changed|
|Button.Subtle Disabled.pressed.chromium.png|25| Changed|
|Button.Subtle Disabled - Dark Mode.pressed.chromium.png|187| Changed|
|Button.Transparent Disabled - Dark Mode.pressed.chromium.png|199| Changed|
|Button.Transparent Disabled - Dark Mode.hover.chromium.png|199| Changed|
|Button.Transparent Disabled.default.chromium.png|27| Changed|
|Button.Transparent Disabled.pressed.chromium.png|27| Changed|

</details>


  <details><summary><b>Checkbox</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Circular - RTL.normal.chromium.png|4733| Changed|
|Checkbox.Circular - Dark Mode.normal.chromium.png|792| Changed|
|Checkbox.Circular Checked - Dark Mode.normal.chromium.png|1080| Changed|
|Checkbox.Circular Indeterminate - Dark Mode.indeterminate.chromium.png|1102| Changed|
|Checkbox.Circular Disabled Checked - Dark Mode.normal.chromium.png|799| Changed|
|Checkbox.Circular Disabled.normal.chromium.png|4711| Changed|
|Checkbox.Circular Disabled Checked.normal.chromium.png|4728| Changed|
|Checkbox.Circular Indeterminate.normal.chromium.png|4786| Changed|
|Checkbox.Default - Dark Mode.normal.chromium.png|814| Changed|
|Checkbox.Circular Indeterminate.indeterminate.chromium.png|4869| Changed|
|Checkbox.Disabled Checked - Dark Mode.normal.chromium.png|777| Changed|
|Checkbox.Checked - Dark Mode.normal.chromium.png|982| Changed|
|Checkbox.Disabled.normal.chromium.png|4705| Changed|
|Checkbox.Disabled Checked.normal.chromium.png|4721| Changed|
|Checkbox.Checked.normal.chromium.png|4855| Changed|
|Checkbox.Disabled - Dark Mode.normal.chromium.png|739| Changed|
|Checkbox.Circular.normal.chromium.png|4734| Changed|
|Checkbox.Indeterminate.normal.chromium.png|4776| Changed|
|Checkbox.Indeterminate - Dark Mode.indeterminate.chromium.png|993| Changed|
|Checkbox.Default.normal.chromium.png|4751| Changed|
|Checkbox.Indeterminate - Dark Mode.normal.chromium.png|877| Changed|
|Checkbox.Circular Disabled - Dark Mode.normal.chromium.png|760| Changed|
|Checkbox.Circular Checked.normal.chromium.png|4883| Changed|
|Checkbox.Default - RTL.normal.chromium.png|4751| Changed|
|Checkbox.Circular Indeterminate - Dark Mode.normal.chromium.png|936| Changed|
|Checkbox.Indeterminate.indeterminate.chromium.png|4847| Changed|

</details>


  <details><summary><b>Divider</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.With Svg - RTL.normal.chromium.png|3| Changed|
|Divider.With Svg.normal.chromium.png|3| Changed|
|Divider.Vertical With Svg.normal.chromium.png|4| Changed|

</details>


  <details><summary><b>Label</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Size Small.normal.chromium.png|661| Changed|
|Label.Required - RTL.normal.chromium.png|11| Changed|

</details>


  <details><summary><b>Link</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link.Default.default.chromium.png|0| Added|
|Link.Default.hover.chromium.png|0| Added|
|Link.Default.pressed.chromium.png|0| Added|
|Link.Inline.default.chromium.png|0| Added|
|Link.Inline.hover.chromium.png|0| Added|
|Link.Inline.pressed.chromium.png|0| Added|
|Link.Subtle.default.chromium.png|0| Added|
|Link.Subtle.hover.chromium.png|0| Added|
|Link.Subtle.pressed.chromium.png|0| Added|
|Link.With Long Text.default.chromium.png|0| Added|

</details>


  <details><summary><b>MenuList</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuList.Checkbox With Icons.1st selected.chromium.png|4| Changed|
|MenuList.Checkbox - RTL.2nd selected.chromium.png|1| Changed|
|MenuList.Checkbox With Icons.normal.chromium.png|4| Changed|
|MenuList.Checkbox With Icons - RTL.1st selected.chromium.png|4| Changed|
|MenuList.Checkbox.2nd selected.chromium.png|1| Changed|
|MenuList.Checkbox With Icons - RTL.normal.chromium.png|4| Changed|
|MenuList.Radio With Icons - RTL.normal.chromium.png|4| Changed|
|MenuList.With Icons.hover menuitem.chromium.png|3| Changed|
|MenuList.Checkbox With Icons - RTL.2nd selected.chromium.png|5| Changed|
|MenuList.Checkbox With Icons.2nd selected.chromium.png|5| Changed|
|MenuList.Radio With Icons - RTL.2nd selected.chromium.png|4| Changed|
|MenuList.Radio With Icons.2nd selected.chromium.png|4| Changed|
|MenuList.Radio With Icons.1st selected.chromium.png|4| Changed|
|MenuList.Radio With Icons - RTL.1st selected.chromium.png|4| Changed|
|MenuList.With Icons.normal.chromium.png|4| Changed|
|MenuList.Radio With Icons.normal.chromium.png|4| Changed|

</details>


  <details><summary><b>Slider</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Size Small.rightArrow.chromium.png|1| Changed|
|Slider.Size Small.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Switch</b> 34 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked - RTL.normal.chromium.png|5132| Changed|
|Switch.Checked - RTL.hover.chromium.png|5150| Changed|
|Switch.Default - Dark Mode.hover.chromium.png|1071| Changed|
|Switch.Checked.normal.chromium.png|5132| Changed|
|Switch.Checked.hover.chromium.png|5150| Changed|
|Switch.Checked - Dark Mode.hover.chromium.png|1169| Changed|
|Switch.Default.hover.chromium.png|5044| Changed|
|Switch.Checked - Dark Mode.normal.chromium.png|1130| Changed|
|Switch.Default - RTL.hover.chromium.png|5044| Changed|
|Switch.Default - RTL.normal.chromium.png|5009| Changed|
|Switch.Disabled - Dark Mode.hover.chromium.png|909| Changed|
|Switch.Default.normal.chromium.png|5009| Changed|
|Switch.Default - Dark Mode.normal.chromium.png|1019| Changed|
|Switch.Disabled - Dark Mode.normal.chromium.png|909| Changed|
|Switch.Disabled - RTL.hover.chromium.png|4914| Changed|
|Switch.Disabled Checked - Dark Mode.hover.chromium.png|1074| Changed|
|Switch.Disabled Checked.normal.chromium.png|5047| Changed|
|Switch.Disabled.hover.chromium.png|4914| Changed|
|Switch.Disabled - RTL.normal.chromium.png|4914| Changed|
|Switch.Disabled Checked.hover.chromium.png|5047| Changed|
|Switch.Disabled Checked - Dark Mode.normal.chromium.png|1074| Changed|
|Switch.Disabled.normal.chromium.png|4914| Changed|
|Switch.Label Above - RTL.hover.chromium.png|0| Removed|
|Switch.Label Above - RTL.normal.chromium.png|0| Removed|
|Switch.Label Above.hover.chromium.png|0| Removed|
|Switch.Label Above.normal.chromium.png|0| Removed|
|Switch.Label After - RTL.hover.chromium.png|0| Removed|
|Switch.Label After - RTL.normal.chromium.png|0| Removed|
|Switch.Label After.hover.chromium.png|0| Removed|
|Switch.Label After.normal.chromium.png|0| Removed|
|Switch.Label Before - RTL.hover.chromium.png|0| Removed|
|Switch.Label Before - RTL.normal.chromium.png|0| Removed|
|Switch.Label Before.hover.chromium.png|0| Removed|
|Switch.Label Before.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Text</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Strikethrough.normal.chromium.png|236| Changed|
|Text.Block.normal.chromium.png|995| Changed|

</details>


  <details><summary><b>TextInput</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextInput.Size Large.normal.chromium.png|3| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>